site/layouts/partials/head/meta-custom.html

```diff
@@ -0,0 +1,50 @@
+<!-- Check for valid date -->
+{{ if not .PublishDate.IsZero }}
+<meta itemprop="datePublished" content="{{ dateFormat "2006-01-02" .Date }}">
+<meta property="article:published_time" content="{{ dateFormat "2006-01-02" .Date }}">
+<meta property="og:updated_time" content="{{ dateFormat "2006-01-02" .Date }}">
+{{ end }}
+<!-- Categories & Tags Present -->
+{{ if or (.Params.Categories) (.Params.tags) }}
+<!-- Categories -->
+{{ if .Params.categories }}
+<meta name="twitter:label1" value="Categories 📂️">
+<meta name="twitter:data1" value="{{ delimit .Params.categories ", " }}">
+{{ end }}
+<!-- Twitter Labels -->
+{{ if .Params.tags }}
+<meta name="twitter:label2" value="Tags 📌️">
+<meta name="twitter:data2" value="{{ delimit .Params.tags ", " }}">
+{{ end }}
+{{ end }}
+<!-- Keywords -->
+{{ if or (.Params.categories) (.Params.tags) }}
+<meta itemprop="keywords" content="{{ if .Params.categories }}{{ delimit .Params.categories ", " }}{{ end }}{{ if .Params.tags }}{{ delimit .Params.tags ", " }}{{ end }}techqueria">
```

Comment:
these params are in config toml?

Reply:
no those only apply to blog posts for the future - they could have tags or categories